portfolio/src/main/webapp/projects.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Chenyu's portfolio</title>
+
+    <!-- Font awesome icons -->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
+
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+
+  </head>
+  <body>
+    <div class="content container-fluid">   
+      <div class="row" id="projects">
+        <!-- the left info bar -->
+        <div class="col-sm-2 info-bar">
+          <div id="avatar">
+            <img src="images/avatar.JPG" alt="Me"/>
+            <img src="images/avatar2.JPG" alt="Me2" class="img-top"/>
+          </div>
+          <p id="name">Chenyu Zhang</p>
+          <div id="location">
+            <i class="fa fa-map-marker" aria-hidden="true" style="font-size:24px"></i>
+            <span>Baltimore, MD</span>
+          </div>
+          <div id="links">
+            <span>Relevant links:</span>
+            <a href="https://www.linkedin.com/in/chenyu-heidi-zhang-jhu/"><i class="fa fa-linkedin-square" style="font-size:24px"></i> link</a>
+            <a href="https://github.com/ChenyuHeidiZhang"><i class="fa fa-github" aria-hidden="true" style="font-size:24px"></i> link</a>
+          </div>
+        </div>
+
+        <!-- main content -->
```

Comment:
Capitalization and "." here and below too :)

Reply:
Done